doc/federation/federation.rst

```diff
@@ -0,0 +1,87 @@
+.. _federation/federation:
+
+========================
+The BinderHub Federation
+========================
+
+While it may seem like ``mybinder.org`` is a single website, it is
+in fact a **federation** of teams that deploy public BinderHubs to serve
+the community. This page lists the BinderHubs that currently help
+power ``mybinder.org``.
+
+Visiting ``mybinder.org`` will randomly redirect you to one
+of the following BinderHubs. You may also directly access a BinderHub
+by using its respective sub-domain in your URL.
+
+If your organization is interested in becoming part of the BinderHub
+federation, check out :ref:`federation/joining`.
+
+.. _federation/federation-list:
+
+Members of the BinderHub Federation
+===================================
+
+Here is a list of the current members of the BinderHub federation:
+
+.. include:: data-federation.txt
+
+
+.. _federation/joining:
+
+Joining the BinderHub Federation
```

Comment:
This title and the next one are "the same", at least they are too similar to make it clear what the difference between them is. Maybe we could call this a FAQ section or "Points to consider when thinking about joining" (needs a shorter version).